### ISSUE TITLE
Clarify which exact SQL syntaxes are supported

### ISSUE THREAD

**Issue by mcandre** [xwb1989/sqlparser#57]
Thank you for providing a tremendously helpful library for managing SQL code! Could you help identify exactly which SQL dialects are supported?

For example, is sqlparser limited to MySQL? Or does it support PostgreSQL? What about MSSQL? SQLite?

And what of the different ISO SQL standards? Is sqlparser up to date with any of the particular ISO SQL standard editions?

Thanks again for providing the open source community with sqlparser, a valuable system.

**Comment by bramp** [comment]
Yes, we should make that clearer. But incase you are asking. It is limited to the MySQL syntax.
